index.html

```diff
@@ -115,16 +129,19 @@
           <div class="hero-logo__img"></div>
           <h1 class="hero-logo__name">UNIHACK</h1>
         </div>
-        <p>Something something good!</p>
-        <p>- CEO of Bosch</p>
+        <div class="quote">
+          <p>Blown away by the great ideas at #UNIHACK2015!</p>
+          <p>Makes judging very difficult</p>
+        </div>
+        <p class="quote-from">- Hadi Micheal (SE at Deloitte)</p>
       </div>
     </section>
     <!-- ABOUT UNIHACK SECTION -->
     <section class="container above-fold">
       <h1>What is UNIHACK?</h1>
       <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Blandit aliquam etiam erat velit scelerisque. Tempus iaculis urna id volutpat lacus laoreet non curabitur gravida. Amet nisl suscipit adipiscing bibendum est ultricies. Semper viverra nam libero justo. Amet massa vitae tortor condimentum. Aliquam etiam erat velit scelerisque in dictum. Nibh praesent tristique magna sit amet purus gravida. Lectus arcu bibendum at varius vel. Pellentesque dignissim enim sit amet venenatis urna. Ut lectus arcu bibendum at varius vel pharetra. Rhoncus dolor purus non enim praesent elementum facilisis leo vel. Purus in massa tempor nec feugiat nisl pretium fusce id. Sed arcu non odio euismod lacinia at quis risus sed. Amet massa vitae tortor condimentum lacinia quis vel eros. Vitae et leo duis ut diam quam nulla porttitor massa. Ultricies leo integer malesuada nunc vel risus.</p>
       <p>Elementum eu facilisis sed odio morbi quis commodo odio aenean. Nunc sed augue lacus viverra vitae. Eleifend donec pretium vulputate sapien nec sagittis. Non nisi est sit amet facilisis magna. Mauris ultrices eros in cursus turpis massa. Nunc sed id semper risus in hendrerit. Integer vitae justo eget magna. In vitae turpis massa sed elementum tempus. Justo eget magna fermentum iaculis eu non diam phasellus. Viverra orci sagittis eu volutpat odio facilisis mauris sit amet. Nuncconsequatinterdum varius sit amet mattis vulputate enim nulla. Condimentum id venenatis a condimentum vitae sapien pellentesque habitant. Consequat mauris nunc congue nisi vitae suscipit tellus mauris. Urna molestie at elementum eu facilisis sed odio morbi.</p>
-      <a target="_blank" rel="noreferrer" href="https://unihack.net/about/" class="btn">Learn More</a>
+      <a target="_blank" rel="noreferrer" href="https://unihack.net/about/" class="btn primary">Learn More</a>
```

Comment:
Minor nit, but if we are following BEM then this should be `btn-primary`.

Fine for now given deadlines, but highlighting it now so we fix this later.